ODIN_II/regression_test/benchmark/task/mixing_optimization/config_file/task.conf

```diff
@@ -0,0 +1,6 @@
+##############################################
+# Configuration file for running experiments
+##############################################
+# Path to directory of circuits to use
+
```

Comment:
add the verilog and the arches here

Reply:
if you want to make this a vtr_flow task i'd be okay, just maybe create a vtr_task rather than task so we can keep things like this tidy

Reply:
Thank you, I will consider improving that